sdk/python/foundation-models/system/docs/component_docs/image_finetune/transformers_finetune_component.md

```diff
@@ -32,7 +32,7 @@ This component enables finetuning of pretrained models on custom or pre-availabl
 
 7. _metric_for_best_model_ (string, optional)
 
-    Specify the metric to use to compare two different models. If left empty, will be chosen automatically based on the task type and model selected. It could be one of [`loss`, `f1_score_macro`, `accuracy`, `precision_score_macro`, `recall_score_macro`, `iou`, `iou_macro`, `iou_micro`, `iou_weighted`].
+    Specify the metric to use to compare two different models. If left empty, will be chosen automatically based on the task type selected. It could be one of [`loss`, `f1_score_macro`, `accuracy`, `precision_score_macro`, `recall_score_macro`, `iou`, `iou_macro`, `iou_micro`, `iou_weighted`].
```

Comment:
Minor: We can add the defaults here such as "accuracy" for image-multiclass classification, "iou" for "image multilabel classification".